tests/unit/helpers.js

```diff
@@ -420,13 +420,14 @@ export function createUserAccountResponse({
 //   ...
 // }
 export const getFakeConfig = (params = {}) => {
-  Object.keys(params).forEach((key) => {
+  // eslint-disable-next-line no-restricted-syntax
+  for (const key of Object.keys(params)) {
```

Comment:
Oh sorry, I missed this comment before but we generally don't use `for...` loops, right? What's wrong with using `Object.keys().forEach()` as before?

Reply:
It might be good for us to get into the habit of `for...of`. It has some benefits:
- You can use it on any iterable, not just arrays
- You can use a `break`

None of this applies to my patch but it seems reasonable.

I'm not really sure what's up with the eslint rule about it. We use `for...of` in a couple other places already and they each disable the lint rule.

Reply:
Fair points!

I believe the rule stemmed more from using `for in` rather than `for of` but I forget exactly why. We'd have to go back and look at the issues to see, @muffinresearch might remember as I believe he made that PR.

Reply:
I did not know the `for of` es6 feature, but while looking for more info, I stumbled upon this issue, which might interest both of you: https://github.com/airbnb/javascript/issues/1271.

Reply:
Yeah, that is a fair point in the issue of `break` being as naughty as a `GOTO`... it's not a lovely flow and if we need to use one maybe we could be writing our code better anyway.

Anyhow, I mostly just like:

* things being functions (`forEach`, `map`, etc.)
* sticking to one way of doing things (we tend to use array methods lots and we can convert things to arrays easily when they aren't already an array, meaning our objects are more predictable)
* I don't like adding code that goes against our lint rules when the reason is 🤷‍♂️ and the other code looks just as readable.

Reply:
I had originally suggested it because it's the standard for the JS I've been writing for Firefox, and one of the justifications for that is that it is apparently faster. I'm not sure how much we care about that, especially in test code, and I agree that having a standard and sticking to it makes sense. If we'd rather stick with forEach, and make that a standard, I'm totally fine with that.

Reply:
Yeah, that kind of micro-optimisation isn't really something I think we care about in the app... I'd say it's better to write readable code over fast code in this instance.

That said `for of` isn't terribly worse for readability, so it's not as big a deal (things like decrementing loops for performance always bothered me, though I forget if that's still a thing).

Agreed on consistency, mostly. Let's just pick one way to do it; I like `forEach` because it seems to encourage side effects less and other reasons I stated, but largely I just want consistency 😄